guides/pd-disaggregation/README.cpu.md

```diff
@@ -0,0 +1,198 @@
+# CPU PD Disaggregation Deployment Guide
```

Comment:
No need to add the README. I will remove the  README.xpu.md in pd example. Merge it in main docs.